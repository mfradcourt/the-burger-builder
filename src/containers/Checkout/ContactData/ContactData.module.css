.ContactData {
  width: 50%;
  margin: 20px auto;
  text-align: center;
  border: 1px solid #ccc;
}
