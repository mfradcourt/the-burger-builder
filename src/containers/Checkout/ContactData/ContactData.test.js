import React from 'react';
import { shallow } from "enzyme";

import ContactData from './ContactData';

describe('<ContactData />', () => {
  let wrapper;

  it('should render ContactData', () => {
    wrapper = shallow(
      <ContactData/>
    );

    expect(wrapper).toMatchSnapshot();
  });
});
