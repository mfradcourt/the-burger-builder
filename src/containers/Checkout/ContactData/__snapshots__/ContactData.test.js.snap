// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ContactData /> should render ContactData 1`] = `
<div
  className="ContactData"
>
  <h4>
    Enter you contact data
  </h4>
  <form
    onSubmit={[Function]}
  >
    <input
      changed={[Function]}
      elementConfig={
        Object {
          "placeholder": "Your Name",
          "type": "text",
        }
      }
      elementType="input"
      invalid={true}
      key="name"
      shouldValidate={
        Object {
          "required": true,
        }
      }
      touched={false}
      value=""
    />
    <input
      changed={[Function]}
      elementConfig={
        Object {
          "placeholder": "Street",
          "type": "text",
        }
      }
      elementType="input"
      invalid={true}
      key="street"
      shouldValidate={
        Object {
          "required": true,
        }
      }
      touched={false}
      value=""
    />
    <input
      changed={[Function]}
      elementConfig={
        Object {
          "placeholder": "ZipCode",
          "type": "text",
        }
      }
      elementType="input"
      invalid={true}
      key="zipCode"
      shouldValidate={
        Object {
          "maxLength": 5,
          "minLength": 5,
          "required": true,
        }
      }
      touched={false}
      value=""
    />
    <input
      changed={[Function]}
      elementConfig={
        Object {
          "placeholder": "Country",
          "type": "text",
        }
      }
      elementType="input"
      invalid={true}
      key="country"
      shouldValidate={
        Object {
          "required": true,
        }
      }
      touched={false}
      value=""
    />
    <input
      changed={[Function]}
      elementConfig={
        Object {
          "placeholder": "Email",
          "type": "text",
        }
      }
      elementType="input"
      invalid={true}
      key="email"
      shouldValidate={
        Object {
          "required": true,
        }
      }
      touched={false}
      value=""
    />
    <input
      changed={[Function]}
      elementConfig={
        Object {
          "options": Array [
            Object {
              "displayValue": "Fastest",
              "value": "fastest",
            },
            Object {
              "displayValue": "Cheapest",
              "value": "cheapest",
            },
          ],
        }
      }
      elementType="select"
      invalid={false}
      key="deliveryMethod"
      shouldValidate={Object {}}
      value="fastest"
    />
    <button
      btnType="Success"
      disabled={true}
    >
      ORDER
    </button>
  </form>
</div>
`;
